docs/index.md

```diff
@@ -4,6 +4,28 @@ Release **{sub-ref}`release`**  ([What's new?](changelog.md))
 
 ```{include} ../README.md
 :start-after: 'teaser-begin -->'
+:end-before: '<!-- sponsor-break-begin'
+```
+
+<!-- [[[cog
```

Comment:
I think, this could be turned into an in-tree Sphinx extension and could be better integrated with tox and other things that call Sphinx. Just an idea for the future.

Reply:
I did some research and it wasn't trivial, so I took the safe route for now. The problem is that I need cog for the README anyways :|

Reply:
Sure. What I meant is that a sphinx extension could still call cog or whatever. It'd just be less copy-pasted things. But that's not for now. I know how to write an extension, but I don't suggest that this is something to be done right in this PR.